### PR TITLE
Rename "Stripping" section to "Host Removal"

### PR DESCRIPTION
**Pull request by olgabot**
While many languages like Python have `str.strip`, `str.rstrip`, `str.lstrip`, it's not always clear to non-computer aficionados what it means. I think the flags make sense in context of their usage text, but I feel "Host Removal" is more clear to a new user. Happy to discuss!

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/eager branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/eager)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --paired_end`).
- [ ] Make sure your code lints ([`nf-core lint .`](https://nf-co.re/tools)).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/eager/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by jfy133** [comment]
One for @maxibor. Do you think it is also worth updating this in script(-name)/process label etc.?

**Comment by apeltzer** [comment]
Guess we should - host-removal also makes it easier to understand, so I'm all for it ;-)

**Comment by jfy133** [comment]
@maxibor thoughts, now you're back from holiday?

**Comment by maxibor** [comment]
Sure, that sounds like a better name. Not too familiar yet with the new json-schema though ;)

**Comment by jfy133** [comment]
@maxibor yes we should change it everywhere in my opinion (including the script itself, if you're not too attached to the name). Do you want to take that over?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -103,9 +103,9 @@ jobs:
       - name: MAPPER_BT2 Test running with BowTie2
         run: |
           nextflow run ${GITHUB_WORKSPACE} -profile test_tsv,docker --mapper 'bowtie2' --bt2_alignmode 'local' --bt2_sensitivity 'sensitive' --bt2n 1 --bt2l 16 --bt2_trim5 1 --bt2_trim3 1
-      - name: STRIP_FASTQ Run the basic pipeline with output unmapped reads as fastq
+      - name: HOST REMOVAL_FASTQ Run the basic pipeline with output unmapped reads as fastq
```
Comment (TCLamnidis) [inline .github/workflows/ci.yml:106]:
```suggestion
      - name: HOST_REMOVAL_FASTQ Run the basic pipeline with output unmapped reads as fastq
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -55,6 +55,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * [#516](https://github.com/nf-core/eager/issues/516) - Made bedtools not report out of memory exit code when warning of inconsistant FASTA/Bed entry names
 * [#504](https://github.com/nf-core/eager/issues/504) - Removed uninformative sexdeterrmine-snps plot from MultiQC report.
 * Nuclear contamination is now reported with the correct library names.
+* [#531](https://github.com/nf-core/eager/pull/531) - Renamed FASTQ stripping with host removal
```
Comment (TCLamnidis) [inline CHANGELOG.md:58]:
```suggestion
* [#531](https://github.com/nf-core/eager/pull/531) - Renamed 'FASTQ stripping' to 'host removal'
```

File: docs/usage.md [hunk 3]
```diff
@@ -1303,7 +1298,7 @@ when left-over sequencing artefacts of in-line barcodes present Default: 0
 Number of bases to trim of 3' (right) end of read prior alignment. Maybe useful
 when left-over sequencing artefacts of in-line barcodes present Default: 0.
 
-### Mapped Reads Stripping
+### Mapped Reads Host Removal
```
Comment (TCLamnidis) [inline docs/usage.md:1301]:
```suggestion
### Removal of Host-Mapped Reads
```

File: docs/usage.md [hunk 4]
```diff
@@ -1318,17 +1313,16 @@ your data to apply their own adapter removal/read merging procedures, while
 maintaining anonyminity for sample donors - for example with microbiome
 research.
 
-If using TSV input, stripping is performed library, i.e. after lane merging.
+If using TSV input, mapped read removal is performed library, i.e. after lane merging.
```
Comment (TCLamnidis) [inline docs/usage.md:1316]:
```suggestion
If using TSV input, mapped read removal is performed per library, i.e. after lane merging.
```

File: nextflow_schema.json [hunk 5]
```diff
@@ -570,24 +570,24 @@
             },
             "fa_icon": "fas fa-layer-group"
         },
-        "stripping": {
-            "title": "Stripping",
+        "host_removal": {
+            "title": "Host Removal",
```
Comment (TCLamnidis) [inline nextflow_schema.json:574]:
```suggestion
            "title": "Removal of Host-Mapped Reads",
```
To sync with title change in `usage.md`

**Review by TCLamnidis — APPROVED** [review]

**Review by maxibor — APPROVED** [review]

**Comment by apeltzer** [comment]
Thanks everyone involved - @olgabot , @maxibor @jfy133 :-)

**Pull request merged**